### PR TITLE
adding signout acceptance test passed

### LINKED ISSUE #10: Sign out from app
As a user
In order to exit the application 
I should be able to sign out from the application

### PR DESCRIPTION
**Pull request by satorinic**
Fixes #10 Sign out from app

### Changes proposed by this PR

Added User logout test - passed

### What did you learn working on this PR?
How to write features to be able to reuse existing step definitions.

### CODE REVIEW

File: features/user_signout.feature [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Feature: As a user
+  In order to exit the application
+  I should be able to sign out from the application
+
+  Background:
+    Given the user "Bob" exist
+    And I am logged in as "Bob"
+    And show me the page
```
Comment (diraulo) [inline features/user_signout.feature]: This step should be removed

**Review by diraulo — CHANGES_REQUESTED** [review]
Looks good otherwise 👍 
Remove that debugging step and I can :shipit:

File: features/user_signout.feature (hunk 1 above)
Comment (j0sv) [reply features/user_signout.feature]: Removed Show me the page

**Comment by satorinic** [comment]
fixes #10

**Review by diraulo — APPROVED** [review]
LGTM 🚀

**Pull request merged**